src/content/en/updates/2017/12/modulepreload.md

```diff
@@ -0,0 +1,147 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: &lt;link rel="modulepreload"&gt; offers a way of declaratively loading JavaScript modules ahead of time.
```

Comment:
Description is only used for the `meta` description on the page, you shouldn't include code or escaped code.

Reply:
Ah, thank you! Fixed.